### PR TITLE
updated checkAttrResponsive, updated checkAttr

### PR DESCRIPTION
**Pull request by volfkarlo**
Changelog:
- updated chapter checkAttrResponsive
- updated checkAttr docs

### CODE REVIEW

**Review by iruzevic — DISMISSED** [review]

File: website/docs/basics/helpers-php.md [hunk 1]
```diff
@@ -50,7 +50,11 @@ This function is the PHP equivalent of the JavaScript function on this [link](he
 
 ## checkAttr
 
-This function is the PHP equivalent of the JavaScript function on this [link](helpers-javascript).
+This function is the really similar to the same JavaScript function on this [link](helpers-javascript). Except there is no last parameter which is used only in wp-admin to detect `undefined` attributes.
```
Comment (goranalkovic-infinum) [inline website/docs/basics/helpers-php.md:53]:
```suggestion
This function is the PHP equivalent of [this JavaScript function](helpers-javascript), except there is no`undefinedAllowed` parameter, which is used only on the WordPress admin side.
```

File: website/docs/basics/helpers-php.md [hunk 2]
```diff
@@ -50,7 +50,11 @@ This function is the PHP equivalent of the JavaScript function on this [link](he
...
+
+## checkAttrResponsive
+
+This function is also the really similar to the same JavaScript function on this [link](helpers-javascript). Except there is no last parameter which is used only in wp-admin to detect `undefined` attributes.
```
Comment (goranalkovic-infinum) [inline website/docs/basics/helpers-php.md:57]:
```suggestion
This function is the PHP equivalent of [this JavaScript function](helpers-javascript), except there is no `undefinedAllowed` parameter - it's used only on the WordPress admin side.
```

File: website/docs/basics/helpers-javascript.md [hunk 3]
```diff
@@ -28,29 +28,31 @@ newSuperTestTitle
 
 ## checkAttr
 
-Checks whether the attributes exist in the attributes list and adds a default value if they don't.
+Checks whether the attributes exist in the attributes list. If value is not set, it checks for default value. If default value is not set, it adds fallback value depending on type, or if `undefinedAllowed` is set to `true`, it sets it to `undefined`.
```
Comment (dingo-d) [inline website/docs/basics/helpers-javascript.md:31]:
```suggestion
Checks whether the attributes exist in the attributes list. If the value is not set, it checks for the default value. If the default value is not set, it adds a fallback value depending on the type, or if `undefinedAllowed` is set to `true`, it sets it to `undefined`.
```

**Review by dingo-d — CHANGES_REQUESTED** [review]
Small grammar fixes

File: website/docs/basics/helpers-php.md (hunk 2 above)
Comment (volfkarlo) [reply website/docs/basics/helpers-php.md:57]: >Last isn't the best word here, as more parameters could be added in the future :)

I fully agree! 👍

**Review by IvanGrginovInf — APPROVED** [review]

**Review by goranalkovic-infinum — APPROVED** [review]

**Review by dingo-d — APPROVED** [review]

**Pull request merged**